doc/release-notes/2.2.md

```diff
@@ -1,10 +1,65 @@
 # 2.2 Release Notes
 
+Version 2.2.0 of the Npgsql Entity Framework Core provider has been released and is available on nuget. This version works with [version 2.2.0 of Entity Framework Core](https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-2.2), and contains some new Npgsql features as well.
+
+This release was result of hard work by [@roji](https://github.com/roji/), [@austindrenski](https://github.com/austindrenski), [@yohdeadfall](https://github.com/yohdeadfall) and [@khellang](https://github.com/khellang).
+
 ## New Features
 
 Aside from general EF Core features new in 2.2.0, the Npgsql EF Core provider contains the following major new features:
 
-* Support for PostgreSQL user-defined ranges. [See the documentation for more details](../mapping/range.md).
+### PostgreSQL 11 covering indexes
+
+PostgreSQL 11 introduced [covering indexes feature](https://paquier.xyz/postgresql-2/postgres-11-covering-indexes), which allow you to include "non-key" columns in your indices. This allows you to perform index-only scans and can provide a significant performance boost. Support has been added in ([#697](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/issues/697)):
+
+```c#
+protected override void OnConfiguring(DbContextOptionsBuilder builder) {
+    modelBuilder.Entity<Blog>()
+        .ForNpgsqlHasIndex(b => b.Id)
+        .ForNpgsqlInclude(b => b.Name);
+}
+```
+
+This will create an index for searching on `Id`, but containing also the column `Name`, so that reading the latter will not involve accessing the table. [See the documentation for more details](../modeling/indices.md).
+
+Thanks to @khellang for contributing this!
```

Comment:
Can you use `[@khellang](.....)` so this links correctly to his profile when published in docfx? 

(Not with code ticks, those are just here for syntax clarity)